datasource/src/main/resources/liquibase/changelog/changelog-0.1.xml

```diff
@@ -86,4 +86,15 @@
         <addNotNullConstraint constraintName="results_nn" tableName="run" columnName="results"/>
     </changeSet>
 
+    <changeSet id="change_repository_id_type" author="nikita_miazin">
+        <delete tableName="run"/>
+        <delete tableName="project"/>
```

Comment:
These two lines will delete all projects in Cromwell Pipeline. I don't think this is a big deal, since we can easily create a new one.
It's not that hard to make a migration that will parse existing `repository` field, but I really don't want to do it.